### ISSUE TITLE
problem with importing CSV data

### ISSUE THREAD

**Issue by olejka2k** [firefly-iii/firefly-iii#2610]
I am running Firefly III version 4.8.1.1

**Description**
If there is duplicate data in CSV - firefly just gives an error. That is an issue because on the same day I can have 2 identical transactions - and they are skipped, thus CSV import is incorrect.

**Extra info**
Heroku platform

**Comment by JC5** [comment]
Yes, this is by design. If Firefly III can't tell the difference between two lines, only one will be imported.

**Comment by olejka2k** [comment]
Well, I understand that it is always better to start from scratch, but I think it will be a good idea to not to import it at all, but to add something like "duplicate" at the end of the description line?
